src/components/ContentWidgets/Header/HeaderEditor.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+
+export const HeaderEditor: React.FC<WidgetEditorProps> = ({
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let header = <h1>{editedContent.header_content}</h1>;
+    if (editedContent.header_priority == 2) {
```

Comment:
same switch case suggestion here too. 

I would use `let header;` then set it within that switch case.